src/main/java/com/github/cameltooling/dap/internal/CamelDebugAdapterServer.java

```diff
@@ -148,18 +147,20 @@ private SetBreakpointsResponse setBreakpointsSync(SetBreakpointsArguments setBre
 						breakpoint.setNodeId(nodeId);
 						connectionManager.updateBreakpointsWithSources(breakpoint);
 						breakpointIds.add(nodeId);
-						if (sourceBreakpoint.getCondition() != null) {
-							connectionManager.getBacklogDebugger().addConditionalBreakpoint(nodeId, CAMEL_LANGUAGE_SIMPLE, sourceBreakpoint.getCondition());
-						} else {
-							connectionManager.getBacklogDebugger().addBreakpoint(nodeId);
-						}
+						addBreakpoint(sourceBreakpoint, nodeId);
 						breakpoint.setVerified(true);
+					} else {
+						breakpoint.setMessage("The Camel debugger cannot find the related id for "+ source.getPath() + " l." + line);
 					}
 				} catch (Exception e) {
-					LOGGER.warn("Cannot find related id for "+ source.getPath() + " l." + line, e);
+					String baseMessage = "The Camel debugger cannot find the related id for "+ source.getPath() + " l." + line;
```

Comment:
The string literals should be in constants or functions.

As these have some values been used, is better to put a closure or function that returns this string

This is also duplicated with line 153 so extra important

Reply:
Good point on the duplication. I modified the message to be more precise. it is slightly different cases and better to have more precise message